example-playbooks/ideal_token_ring/README.md

```diff
@@ -0,0 +1,25 @@
+# Use ideal token ring for empty non initialized cluster
+
+This is an ansible playbook that lets you assign ideal token ring to a known size cluster
```

Comment:
And where this cluster would get from?
I had a very different patchwork in mind: instrument a role to get tokens for each host in some variable and feed it that variable via a config file.

This file can be generated by a Jenkins task in a run-time if requested - hence the support for inventory files I added to the script.